dbt_platform_helper/providers/ecs.py

```diff
@@ -84,4 +83,16 @@ def ecs_exec_is_available(self, cluster_arn: str, task_arns: List[str]):
                 time.sleep(1)
 
         if execute_command_agent_status != "RUNNING":
-            raise ECSAgentNotRunning
+            raise ECSAgentNotRunningException
+
+
+class ECSAgentNotRunningException(AWSException):
```

Comment:
I wonder if the AWSException belongs in platform_exception.py since it (as far as I can see) is just used as a base class. Then this provider wouldn't depend on the aws provider.

Reply:
Good spot.

These should probably be based off an ECSException is we're following this patten. I'll do that now.

Reply:
Done